src/assets/reset.css

```diff
@@ -171,3 +171,13 @@ input,
 select {
   vertical-align: middle;
 }
+
+/* we don't use all:unset, because this removes the outline on :focus:visible */
+button {
+  background: none;
+  color: inherit;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+}
```

Comment:
Might it be easier to do:

```
button {
  all: unset;

  &:focus-visible {
    // explicitly set expected state
  }
}
```

Just a thought. Unsure if that'd be better / if it'd work.